### ISSUE TITLE
Count kills if you left an armed nuke behind

### ISSUE THREAD

**Issue by singalen** [chaosforgeorg/doomrl#19]
It makes no sense not to count the kills of monsters if you left them an armed nuke.

**Comment by OmegaTyrant** [comment]
This suggestion can't be implemented, for the simple reason that it would completely trivialize the Quartermaster Gold, Diamond, and Angelic badges, as well as Eagerness Angelic, if you were able to just exist every floor immediately before the nuke went off and still have the nuke's kills count.

**Comment by brisbang** [comment]
Agreed - I was thinking about this too. Kill counts would be crushed. The skill is in nuking and managing to stay around. Can we close this issue?
